rockspec/lua-resty-hmac-ffi-1.0-0.rockspec

```diff
@@ -0,0 +1,24 @@
+package = 'lua-resty-hmac-ffi'
+version = '1.0-0'
```

Comment:
Any reason to choose a different version number from the internal version (i.e. the same number used when publishing to OPM)?

Reply:
you can just release a new version which name is `v1.0` or create a new tag `v1.0`.
they are the same.

Reply:
> Any reason to choose a different version number from the internal version

not support this in `luarocks`, because the `lua-resty-string` is shipped by openresty, the version is not controlled under luarocks.